src/reducers/entities/groups.js

```diff
@@ -148,6 +148,13 @@ function groups(state = {}, action) {
         }
         return nextState;
     }
+    case GroupTypes.RECEIVED_GROUPS_ASSOCIATED_TO_TEAM: {
```

Comment:
can you add `case GroupTypes.RECEIVED_GROUPS_ASSOCIATED_TO_CHANNEL:` as well, so groups will get populated for that method too?

Reply:
Sure, I'll close https://mattermost.atlassian.net/browse/MM-16531 and do it in this story.